lib/SILOptimizer/Mandatory/MandatoryInlining.cpp

```diff
@@ -404,17 +404,22 @@ static SILFunction *getCalleeFunction(
     return nullptr;
   }
 
-  // If CalleeFunction is a declaration, see if we can load it. If we fail to
-  // load it, bail.
-  if (CalleeFunction->empty()
-      && !AI.getModule().linkFunction(CalleeFunction, Mode))
-    return nullptr;
-
   // If the CalleeFunction is a not-transparent definition, we can not process
   // it.
   if (CalleeFunction->isTransparent() == IsNotTransparent)
     return nullptr;
 
+  // If CalleeFunction is a declaration, see if we can load it.
+  if (CalleeFunction->empty()) {
+    // FIXME: Remove 'Mode'
+    if (Mode != SILOptions::LinkingMode::LinkNone)
+      AI.getModule().loadFunction(CalleeFunction);
```

Comment:
Did you add the API loadFunction? I dislike that we have linkFunction and load function. I would rather have something more explicit about the transitive nature in the function names.

Reply:
I don't like the word "link" for what this code is doing at all, since its not really related to linking in the traditional sense. However, I'd like to punt this and the other renaming to my PR that removes LinkNone. Is that fair?